package-lock.json

```diff
@@ -0,0 +1,2339 @@
+{
```

Comment:
Are you using npm 5?

This file shouldn't be checked in. The platform will ignore it, for one thing.